test/utils/update_resources.go

```diff
@@ -21,7 +21,10 @@ import (
 	"time"
 
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/kubernetes/pkg/kubectl"
+
+	scaleclient "k8s.io/client-go/scale"
```

Comment:
the convention is that if you you introduce aliases, those are suffixes of the path (there are expections coming from long time ago, but we're trying to change that).

So, if any clientgoscale?
But I think you don't need any alias - you don't seem to have "scale" name anywhere in this file.

Reply:
alias removed